src/lib/Components/Commands/exports/Native/SetupRemoteDocker.ts

```diff
@@ -31,6 +31,17 @@ export class SetupRemoteDocker implements Command {
   get generableType(): GenerableType {
     return GenerableType.SETUP_REMOTE_DOCKER;
   }
+
+  /**
+   * Enable docker image layer caching
+   * @param {boolean} enabled - If true, docker layer caching is enabled for the job.
```

Comment:
```suggestion
   * @param enabled - If true, docker layer caching is enabled for the job.
```

In typescript, the type is automatically inferred here.